utils/src/lib.rs

```diff
@@ -1205,9 +1184,9 @@ mod tests {
         );
     }
 
-    /// Executes the _fourbyte_ request and if the site is not down (502 Bad Gateway) executes the
-    /// test
-    async fn test_if_fourbyte_not_down<Req, Out, Test>(r: Req, test: Test)
+    /// Executes the _decode selector_ request and if the site is not down (502 Bad Gateway)
+    /// executes the test
+    async fn test_if_sig_database_not_down<Req, Out, Test>(r: Req, test: Test)
```

Comment:
this function is necessary anymore as this was tailored explicitly for fourbyte, so I think we should just get rid of this.

Reply:
removed; see #1676